doc/debugging.rst

```diff
@@ -14,33 +14,91 @@ This allows breakpoints and the execution flow to be viewed and controlled right
 Debugging with ipdb
 ===================
 
-Spyder offers the following debugging features integrated into the native GUI:
+You can fully control debugger execution from the :guilabel:`Debug` menu, :guilabel:`Debug toolbar` and via configurable keyboard shortcuts, along with the standard ``ipdb`` `console commands`_.
+Additionally, the  :doc:`editor` shows the line of code the debugger is currently stopped on with an arrow.
 
-* Multiple means of setting and clearing normal and conditional breakpoints for any line in a file opened in the :doc:`editor`.
+.. _console commands: https://pythonconquerstheuniverse.wordpress.com/2009/09/10/debugging-in-python/
 
-  * By selecting the respective option from the Debug menu.
-  * Through pressing a configurable keyboard shortcut (:kbd:`F12` for normal, or :kbd:`Shift-F12` for conditional breakpoints by default).
-  * By double-clicking to the left of the line number in an open file.
-  * With an ``ipdb.set_trace()`` statement in your code (after ``import`` ing ``pdb``).
-  * Interactively, using the ``b`` command in an ``ipdb`` session.
+GIF: Inset of Spyder's Editor, with a breakpoint set and the condition dialog open (commands from the toolbar)
+
+Spyder's debugger offers syntax highlighting, code completion and command history, which work exactly like they do in the normal interactive interpreter.
```

Comment:
This was how it read originally (minus the typo), but I proposed a the change to the current version which Juanita accepted, because that phrasing really isn't accurate. Debugging still occurs in the IPython console pane, so it doesn't make sense to say that those features work the same they do in the IPython console they still _are_ in the IPython console, and saying that implies that the debugger is in a separate pane from the console, which is potentially confusing and misleading to users.

That said, I'm not entirely happy with "normal interactive interpreter" in terms of being easily understandable, so (as I mentioned in my original comment) I'd encourage your suggestions if you have a clearer term to use here.

Reply:
Ok, you're right. I don't have other proposals, so let's leave it like this.

Reply:
Okay, sounds good